rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl

```diff
@@ -0,0 +1,103 @@
+## rna-collect-rna-seq-metrics.wdl
+##
+## This workflow runs Picard's CollectRnaSeqMetrics on a BAM file
+## (https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.0.0/picard_analysis_CollectRnaSeqMetrics.php).
+##
+## Inputs:
+## - Per-sample:
+##   - Sample name - Name of the sample associated with the BAM - used in names of outputs.
+##   - BAM file - Path to a BAM file.
+##   - Ref flat file - Gene annotations in refFlat form.
+##   - Ribosomal intervals file - Location of rRNA sequences in genome, in interval_list format.
+##
+## - VM configuration
+##   - Docker image url
+##   - VM disk size
+##   - VM memory
+##   - Num CPU cores
+##   - Runtime zones, ex: "us-central1-a us-central1-b"
+##   - Number of times to try the workflow with a preemptible VM before
+##     falling back to a full-price VM.
+##
+## Outputs:
+## - Per-sample:
+##   - <sample-id>.RNA_Metrics
+
+workflow RNACollectRnaSeqMetrics {
+
+  String sample_name
+  File bam_file
```

Comment:
Add a blank line after this one to separate the per-sample inputs from the reference inputs.